src/v8/v8.cc

```diff
@@ -503,7 +504,7 @@ bool V8::getWord(uint64_t pointer, Word *word) {
   }
   uint32_t word32;
   ::memcpy(&word32, memory_->data() + pointer, size);
-  word->u64_ = wasmtoh(word32);
+  word->u64_ = wasmtoh(word32, usesWasmByteOrder());
```

Comment:
Nit: This could be hardcoded to `true` (same below and in other WasmVMs).

Reply:
hardcoded to true